Testnet/DAOContract/DAOContract/DAOContract.cs

```diff
@@ -224,33 +229,28 @@ public void WhitelistAddresses(byte[] addresses)
 
     public void Deposit()
     {
-        EnsureOwnerOnly();
         Log(new FundRaisedLog { Sender = Message.Sender, Amount = Message.Value });
     }
 
     public override void Receive() => Deposit();
 
-    public void TransferOwnership(Address newOwner)
+    public void ClaimNewOwnership(Address newOwner)
     {
         EnsureOwnerOnly();
-        Owner = newOwner;
+        ClaimedOwner = newOwner;
     }
```

Comment:
Any reason fo the name change? This is a nit pick but ClaimNewOwnership sounds like what we were suggested to do in our audit. 

Certik suggested pull ownership rather than push. So CurrentOwner sets NewOwner. NewOwner calls to accept ownership. I'm not sure if that's what you're going for here or not. 

It seems that you are but you may have things backwards. 

**ApproveOwnership** woul dhave the current owner set a new owner property.
**ClaimOwnership** would check `Message.SEnder == NewOwner` then set `ClaimedOwner = NewOwner` and `NewOwner = Address.Zero`

Reply:
Yes i do what is suggested from Certik. 

I think there is a mistake in suggested code. Actual logic has to be 
ClaimOwnership has to set before new owner claimed by Owner 

Owner = ClaimedOwner; 
ClaimedOwner = Address.Zero

Reply:
It should be this way I think.

1. `Owner` approve `NewOwner`
``` C#
public void ApproveOwner(Address newOwner) 
{
    EnsureOwnerOnly();

    NewOwner = newOwner;
}
```
2. `NewOwner` claims ownership
``` C#
public void ClaimOwnership() 
{
    Assert(Message.Sender == NewOwner);

    Owner = Message.Sender;

    NewOwner = Address.Zero;
}
```